backend/src/graphql/arg/SearchUsersArgs.ts

```diff
@@ -11,9 +12,6 @@ export default class SearchUsersArgs {
   @Field(() => Int, { nullable: true })
   pageSize?: number
```

Comment:
```suggestion
  pageSize?: number | null
```

And why it isn't here then? 👇🏼

Reply:
No. It can never be null. It has a default value instead.

Reply:
Why?
It says it's nullable at line `11`. @Mogge

Reply:
same same